evap/evaluation/templates/profile.html

```diff
@@ -6,17 +6,44 @@
     {{ block.super }}
     <h3>{% trans 'Profile' %}</h3>
 
-    <div class="row mb-3">
-        <div class="col-6">
-            <div class="card">
-                <form id="profile-form" method="POST" class="form-horizontal multiselect-form">
-                    {% csrf_token %}
+    <form id="profile-form" method="POST" class="form-horizontal multiselect-form">
+        {% csrf_token %}
+        <div class="row row-cols-1 row-cols-md-2 g-4 mb-3">
+            <div class="col">
+                <div class="card h-100">
                     <div class="card-body">
                         <h4 class="card-title mb-4">{% trans 'Personal information' %}</h4>
-                        {% include 'bootstrap_form.html' with form=profile_form %}
-                        {% if user.is_editor %}
-                            <table class="table table-vertically-aligned table-borderless">
-                                <tr class="border-bottom">
+                        {% for field in profile_form %}
+                            {% if field != profile_form.delegates %}
+                                {% include 'bootstrap_form_field.html' with field=field %}
+                            {% endif %}
+                        {% endfor %}
+                    </div>
+                </div>
+            </div>
+            {% if user.is_editor %}
+                <div class="col">
+                    <div class="card h-100">
+                        <div class="card-body">
+                            <h4 class="card-title mb-4">{% trans 'Delegates and emails' %}</h4>
+                            <div class="form-text">{% trans 'Your delegates will receive emails sent to you in CC, will be able to prepare evaluations on your behalf, and will see the results of your evaluations.' %}</div>
```

Comment:
I found it a bit odd that the help text for the display name is below the field, but this help text is below a heading / nothing. From a quick glance, maybe moving it below the select element would be more consistent? What do you think?